corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/api.js

```diff
@@ -11,6 +11,7 @@ hqDefine("cloudcare/js/formplayer/menus/api", function () {
         FormplayerFrontend = hqImport("cloudcare/js/formplayer/app"),
         formplayerUtils = hqImport("cloudcare/js/formplayer/utils/utils"),
         ProgressBar = hqImport("cloudcare/js/formplayer/layout/views/progress_bar");
```

Comment:
change ending to a comma to get rid of those errors

Reply:
Changed